lib/amorail/entity/finders.rb

```diff
@@ -52,7 +52,7 @@ def load_record(id)
         remote_url('list'),
         id: id
       )
-      handle_response(response, 'load')
+      handle_response(response, 'load') || nil
```

Comment:
This method is used in [`Entity#push`](https://github.com/teachbase/amorail/blob/master/lib/amorail/entity.rb#L100). And `Entity#push` is used in `Entity#save` and `Entity#update`.

I thought that returning `nil` from `save` and `update` would be wrong.